docs/sources/icd11foundation.md

```diff
@@ -9,6 +9,7 @@ This data source in particular is the ICD11 foundation, not one of its lineariza
 
 **Homepage:** https://icd.who.int/
 
+<<<<<<< HEAD
```

Comment:
Merge error

Reply:
Glad you noticed that. I thought I'd resolved all the conflicts in that file. Fixed!